osquery/tables/system/darwin/password_policy.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <OpenDirectory/OpenDirectory.h>
+#include <vector>
+
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+#include <osquery/utils/conversions/darwin/cfdictionary.h>
+
+namespace osquery {
+namespace tables {
+
+QueryData genPasswordPolicy(QueryContext& context) {
+  QueryData results;
+
+  CFErrorRef error = nullptr;
+  // Create a node of type LocalNodes with the default OpenDirectory Session
+  auto node = ODNodeCreateWithNodeType(
+      kCFAllocatorDefault, kODSessionDefault, kODNodeTypeLocalNodes, &error);
+  if (node == nullptr) {
+    VLOG(1) << "Error creating an OpenDirectory node";
```

Comment:
How often does this occur? It sounds like enough of an error that it should be more than verbose. Please also include the table name